packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -168,7 +172,7 @@ module.exports = function (webpackEnv) {
         // So adding the option here in replacement as per
         // https://github.com/webpack-contrib/css-loader/tree/v3.4.2#onlylocals
         loader: require.resolve('css-loader'),
-        options: { ...cssOptions, onlyLocals: true },
+        options: cssOptions,
```

Comment:
This change wasn't related to our work - looks like a miss from the v9 upgrade. `onlyLocals` is a v3 option for `css-loader`, and v9 upgrades to v4. Even so, you can't pass the option here:

```
ValidationError: Invalid options object. CSS Loader has been initialized using an options object that does not match the API schema.
  - options has an unknown property 'onlyLocals'. These properties are valid:
    object { url?, import?, modules?, icss?, sourceMap?, importLoaders?, esModule? }
```

Reply:
Good spot! In which case I may need to review this and make sure the right exportOnlyLocals is set to true inside `modules`